client/package.json

```diff
@@ -4,10 +4,8 @@
   "license": "MIT",
   "repository": "https://github.com/chanzuckerberg/cellxgene",
   "scripts": {
-    "build":
```

Comment:
this looks like a ws change. does it need to be here?

Reply:
Slipped that in there because the build was breaking. I should be able to get rid of this if I rebase with master, I think Colin fixed the problem.